packages/experiments/src/components/TilesList/TilesList.tsx

```diff
@@ -543,7 +543,7 @@ export class TilesList<TItem> extends React.Component<ITilesListProps<TItem>, IT
 
     if (pageSpecificationCache.byIndex[startIndex]) {
       // If the page specification has already been calculated, return it.
-      // List recalculates all pages if any input changes, so this memoization
+      // List recalculates all pages if any input changes, so this memorization
```

Comment:
This was correct previously. https://en.wikipedia.org/wiki/Memoization
```suggestion
      // List recalculates all pages if any input changes, so this memoization
```